docs/planning.md

```diff
@@ -0,0 +1,81 @@
+# Release and Roadmap Process
+
+We follow a coordinated approach to plan improvements of Eclipse Tractus-X. see also [Open Planning and Refinement](docs/open-refinement-and-planning.md).
+
+While every repository in the [eclipse-tractusx](https://github.com/eclipse-tractusx) GitHub organization
+has its own issue management, the [Release Planning Board](https://github.com/orgs/eclipse-tractusx/projects/26)
+is used to align the overarching Tractus-X releases.
+
+### How can I get involved
+
+In case you experienced a bug, unexpected behaviour, or you want to propose enhancements to Eclipse Tractus-X,
+feel free to use one of the provided [issue templates](https://github.com/eclipse-tractusx/sig-release/issues/new/choose) and describe your request.
+Please be aware, that not every feature request can be integrated and that we also cannot treat every issue with the highest priority.
+
+Every Release planning will be kicked off by two public alignment sessions. The dates and further details will be shared via
+[tractusx-dev](https://accounts.eclipse.org/mailing-list/tractusx-dev) mailing list.
+In addition to that, you can also find public meetings and info about how to join on our
+[Open Meetings](https://eclipse-tractusx.github.io/community/open-meetings) community page.
+Issues or bug reports, that should be discussed in these meetings, have to be opened prior to the meeting via
+our [issue templates](https://github.com/eclipse-tractusx/sig-release/issues/new/choose).
+
+### What can I expect
+
+We really welcome every contribution. Every Bug report and feature proposal takes time to prepare,
+is valuable to our project, and we very much appreciate this input.
+We are giving our best to give a first feedback in one week.
+If we should miss that, please stick with us and just use the commenting function to remind us of the issue.
+
+### Issue structure
+
+Our issues do have important properties, that enable our planning process. These are:
+
+- __Labels:__ We use them to indicate the involved teams (kit or foss component). A label for each involved component is added to an issue
```

Comment:
KIT and FOSS in capitals

Reply:
It is common practice, that GitHub issue label text is written in small letters. But maybe we could make them bold or italic to clarify, that actual labels are meant? Or maybe link to [available labels](https://github.com/eclipse-tractusx/sig-release/labels) directly?

Reply:
fine with small letters